source/open-definition-revision-process-dev.markdown

```diff
@@ -2,19 +2,26 @@
 Definition Revision Approval Process _- draft_
 ====================================
 
-1. Issues with the current Open Definition are identified.  The Open Definition community will disucss on the mailing list and decide which issues are to be dealt with in the next release reach consensus on each of these issues.
```

Comment:
hayzlee13@

Reply:
This step just describes the process starts.  People discuss issues on the mailing list and agree informally about what needs to happen.